crates/biome_cli/tests/commands/lint.rs

```diff
@@ -992,6 +992,7 @@ fn fs_error_unknown() {
 //     ├── symlink_testcase1_1 -> hidden_nested
 //     └── symlink_testcase2 -> hidden_testcase2
 #[test]
+#[ignore = "It regresses on linux since we added the ignore crate, to understand why"]
```

Comment:
@ematipico I see you added a similar ignore in `commands/check.rs`, but it looks like this one in `lint` can also fail on linux, so I'm adding the same one here, unless you think there's another solution or this is actually something incorrect (but since this is just a formatter change, i don't imagine it is).

Reply:
Thank you!